### ISSUE TITLE
find a remedy for Conda-only tools in Conda-less Galaxies

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#3257]
The current situation is that if you don't have Conda (which is optional and a config option) and you try to install Conda-only tool (which are some now, including major ones like fastqc) you will be presented with an installation dialogue completely ignoring dependencies. You are not even notified about this situation during the installation. After the tool is installed (without any deps) it will **not run** or run with **random binary from PATH**. This is bad.

Basically you are forced to somehow provide the binary manually and you are losing reproducibility. 

We should do something with it, for 16.10. At least a warning and info about how to enable Conda.

Thoughts? @mvdbeek @jmchilton @bgruening

**Comment by bgruening** [comment]
We can easily detect if no tool_dependency file is included, correct? In this case we can include a warning or a info box with the link to the conda FAQ?

**Comment by mvdbeek** [comment]
> We can easily detect if no tool_dependency file is included, correct? In this case we can include a warning or a info box with the link to the conda FAQ?

you'd also have to check if there is a package requirement, but yes, that's doable, I think.

**Comment by mvdbeek** [comment]
![wip](https://cloud.githubusercontent.com/assets/6804901/20859709/a8dbae92-b966-11e6-8f1c-785846cacf45.png)

I'm making some progress in this direction, and while I agree this is a major problem, we've had this issue since 16.07 and/or the first time we've decided to go conda-only and it's hardly a bug, it's a missing feature.

**Comment by martenson** [comment]
@mvdbeek As I see it in the start it was a missing feature but with more and more tools going Conda-only this entered the bug-zone for me as it results in huge complications for deployers without any hint of help from our side.

That said, we will surely fix it for 16.10, so nurries. :)
